doc/source/commands/addto.rst

```diff
@@ -1,4 +1,5 @@
 Addto
 -----
+
 .. automodule:: papis.commands.addto
 
```

Comment:
Just making the `commands` docs consistent with a linebreak under header. Very unimportant.

Reply:
Love it!